devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -26,14 +26,14 @@ metadata {
 		capability "Health Check"
 		capability "Sensor"
 
-		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-S", deviceJoinName: "Water Leak Sensor"
-		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315"
-		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-Seu", deviceJoinName: "Water Leak Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-L", deviceJoinName: "Iris Smart Water Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-G", deviceJoinName: "Centralite Water Sensor"
-		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "SmartThings", model: "moisturev4", deviceJoinName: "Water Leak Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "water", deviceJoinName: "Water Leak Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "Sercomm Corp.", model: "SZ-WTD03", deviceJoinName: "Sercomm Water Leak Detector"
+		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-S", deviceJoinName: "SmartSense Water Leak Sensor" //Water Leak Sensor
```

Comment:
Is `SmartSense` the approved name instead of `SmartThings`?

Updates are required to `messages.properties` as well

Reply:
I have no idea, I guess this is a question for Jason.

Reply:
| Incluster                          | Model      | Old DJN                 | New DJN                       |
|------------------------------------|------------|-------------------------|-------------------------------|
| 0000,0001,0003,0402,0500,0020,0B05 | 3315-S     | Water Leak Sensor       | **CentraLite** Water leak sensor  |
| 0000,0001,0003,0402,0500,0020,0B05 | 3315-Seu   | Water Leak Sensor       | **CentraLite** Water leak sensor  |
| 0000,0001,0003,0020,0402,0500,0B05 | 3315-L     | Iris Smart Water Sensor | **Iris** Water leak sensor        |
| 0000,0001,0003,0020,0402,0500,0B05 | 3315-G     | Centralite Water Sensor | **Centralite** Water leak sensor  |
| 0000,0001,0003,000F,0020,0402,0500 | moisturev4 | Water Leak Sensor       | **SmartThings** Water leak sensor |
| 10,003,002,004,020,500             | water      | Water Leak Sensor       | **SmartThings** Water leak sensor |